src/Columns/BooleanColumn.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Backpack\CRUD\Columns;
+
+class BooleanColumn extends Column
+{
+    protected $type = 'boolean';
+
+    /**
+     * @param $yesText
+     * @param $noText
+     * @return $this
+     */
+    public function texts($yesText, $noText)
+    {
+        $this->data['options'] = [
+            0 => $yesText,
```

Comment:
`0, 1` vs `true, false`?

True and false are somewhat more robust are they not?